droughtpipeline/load.py

```diff
@@ -0,0 +1,872 @@
+            if lead_time in triggered_lead_times:
+                shutil.copy(
+                    drought_extent.replace(".tif", f"_{lead_time}.tif"), drought_extent_new
+                )
+                
+            else:
+                logging.info(f"no trigger for lead time {lead_time}")
+                #shutil.copy(drought_extent.replace(".tif", f"_empty.tif"),  drought_extent_new, )
+            self.rasters_sent.append(drought_extent_new)
+            files = {"file": open(drought_extent_new, "rb")}
+            self.ibf_api_post_request( "admin-area-dynamic-data/raster/drought", files=files )
+
+
+
+        # send empty exposure data
+        if len(processed_pcodes) == 0:
+            indicators = [
+                "population_affected",
+                "population_affected_percentage",
+                #"alert_threshold",
+                "forecast_trigger",
+                "forecast_severity",
+
+            ]
+            for indicator in indicators:
+                for adm_level in forecast_data.adm_levels:
+                    exposure_pcodes = []
+                    for pcode in forecast_data.get_pcodes(adm_level=adm_level):
+                        if pcode not in processed_pcodes:
+                            amount = None
+                            if indicator == "population_affected":
+                                amount = 0
+                            elif indicator == "population_affected_percentage":
+                                amount = 0.0
+                            #elif indicator == "alert_threshold":
+                            #    amount = 0.0
+                            elif indicator == "forecast_trigger":
+                                amount = 0
+                            elif indicator == "forecast_severity":
+                                amount = 0
+                            exposure_pcodes.append(
+                                {"placeCode": pcode, "amount": amount}
+                            )
+                    body = {
+                        "countryCodeISO3": country,
+                        "leadTime": "1-day",  # this is a specific check IBF uses to establish no-trigger
+                        "dynamicIndicator": indicator,
+                        "adminLevel": adm_level,
+                        "exposurePlaceCodes": exposure_pcodes,
+                        "disasterType": disasterType,
+                        "eventName": None,  # this is a specific check IBF uses to establish no-trigger
+                        "date": upload_time,
+                    }
+                    self.ibf_api_post_request(
+                        "admin-area-dynamic-data/exposure", body=body
+                    )
+
+        # close events: event/close-events
+        body = {
+            "countryCodeISO3": country, 
+            "disasterType": disasterType,
+            "date": upload_time,
+        }
```

Comment:
this body (and all the commented out text around here) here can be removed. The change you made is good.

Reply:
check!